aries-site/src/data/structure.js

```diff
@@ -355,14 +387,18 @@ export const structure = [
     seoDescription:
       'Footer is a Box with a set of preset properties. Box properties allow you to customize the footer.',
     sections: [],
+    previewComponent: {
+      component: () => <FooterPreview />,
+      justify: 'end',
+    },
   },
   {
     name: 'Grid',
     description:
       'The Grid component is used to layout content. Responsive grid is important to consider in every use case.',
-    previewImage: {
-      src: '/grid-light.svg',
-      alt: 'HPE Grid Preview',
+    previewComponent: {
+      component: () => <GridPreview />,
+      cardColor: 'background-front',
     },
```

Comment:
Wondering if we can collapse the `previewImage` and `previewComponent` into a single `preview` property? These objects could then look like this:
```
{
  name: 'Grid',
  description: 'dfasdfadsf fsdafdsadff fdsafda',
  ...
  preview: {
    component: () => <GridPreview />,
    background: 'background-front',
    image: {
       src: '/grid-light.svg',
       alt: 'HPE Grid Preview',
    },
    justify: 'efadfnd'
  },
},
```

Reply:
I like this idea -- I would assume a preview would have either a component or an image (not both) -- and we could check for that in ContentCard.

Reply:
^ My comment can be ignored, looks like Brittany already implemented it nicely : )